Plugins/DotNET/DotNETExports.cpp

```diff
@@ -30,16 +30,16 @@ struct AllHandlers
 };
 static AllHandlers s_handlers;
 
-static uint32_t s_pushedCount = 0;
+static int32_t s_pushedCount = 0;
```

Comment:
This was always implicitly cast on line 177/175 `cmd->ExecuteCommand(id, s_pushedCount);`